src/components/calcite-list/calcite-list.scss

```diff
@@ -6,7 +6,8 @@
   @apply bg-transparent
   box-border
   flex
-  flex-col;
+  flex-col
+  w-full;
```

Comment:
@asangma changed this so that the list spans all available space instead of just the space it needs.